docs/user_guide/bank/getting_started_customization.md

```diff
@@ -54,18 +54,23 @@ There is also a check-box on the Partner that must be checked for the Partner to
 
 The text of this email will be:  
 
+<blockquote>
 Hello [Partner's name],
-
+<br/>
+<br/>
 This is a friendly reminder that [Your bank's name] requires your human essentials requests to be submitted by [the deadline date, including month and year]
 if you would like to receive a Distribution next month.
-
+<br/>
+<br/>
 Please log into Human Essentials at https://humanessentials.app before this date and submit your request if you are intending to submit an essentials request.
-
+<br/>
+<br/>
 Please contact [Your bank's name] at <%= @organization.email %>
```

Comment:
Ok -- I didn't realize we still had something so ... codey... in this.

Please change  <%= @organization.email %>  to [ Your bank's email]